guides/common/modules/proc_synchronizing-the-satellite-tools-repository.adoc

```diff
@@ -2,7 +2,7 @@
 = Synchronizing the {project-client-name} Repository
 
 Use this section to synchronize the {project-client-name} repository from the Red Hat Content Delivery Network (CDN) to your {Project}.
-This repository provides the `katello-agent`, `katello-host-tools`, and `puppet` packages for clients registered to {ProjectServer}.
+This repository provides the `katello-host-tools` and `puppet` packages for clients registered to {ProjectServer}.
```

Comment:
It provides more than that, so I don't know why we call these out specifically. The repo has openscap client packages, tracer, REX..

Reply:
How about just something like

```suggestion
This repository provides several packages for installation on hosts registered to {ProjectServer}.
```